### PR TITLE
Removed redundant conditionals and variable in basic.py

### PR DESCRIPTION
**Pull request by alexwholland**
##### SUMMARY

- Removed two `if` statements that always evaluated to `True` in the `_set_internal_properties`  method.
- Removed the `argument_spec` variable as it is never used.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

`basic.py`

### CODE REVIEW

**Comment by jborean93** [comment]
@Qalthos are you able to verify that networking no longer uses this function?

@alexwholland are you able to fix the sanity errors.

**Comment by alexwholland** [comment]
> @alexwholland are you able to fix the sanity errors.

Hi @jborean93 What I can gather from the sanity errors is that I need to remove the empty line 1252. I'm unsure if there is something else that I should fix for my changelog or in `basic.py`.

File: changelogs/fragments/76639-remove-redundant-variable-and-conditional.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+bugfixes:
+  - basic - Removed ``if`` statements in ``_set_internal_properties`` that always evaluated to True. 
+Also remove unused variable ``argument_spec``.
```
Comment (jborean93) [inline changelogs/fragments/76639-remove-redundant-variable-and-conditional.yaml]:
```suggestion
bugfixes:
- >-
  basic - Removed ``if`` statements in ``_set_internal_properties`` that always evaluated to True
  and also remove unused variable ``argument_spec``.
```

**Comment by jborean93** [comment]
Looks like it's just your changelog fragment, I've added a suggestion to fix it.

**Pull request closed without merging**